packages/core/src/browser/about-dialog.tsx

```diff
@@ -0,0 +1,76 @@
+/********************************************************************************
+ * Copyright (C) 2018 Ericsson and others.
```

Comment:
[minor] please update to 2020 since it is a new file.

Reply:
about-dialog.ts isn't from dialog.ts... it's renamed from about-dialog.ts... It only looks new because it's mostly rewritten so git couldn't detect it. Did you mean another file perhaps?